core/bones/recordBone.py

```diff
@@ -55,7 +55,11 @@ def parseSubfieldsFromClient(self) -> bool:
 
 	def singleValueFromClient(self, value, skel, name, origData):
 		usingSkel = self.using()
-		usingSkel.fromClient(value)
+
```

Comment:
Nope. Always return all errors. This will suppress warnings and "invalidates other" errors. Error-Handling is done outside (in fromClient).